configuration/integrations/overview.mdx

```diff
@@ -26,6 +26,7 @@ Integrations are a beta feature, so you may not see them in your account yet. As
 | Firebase (Authentication)                                                                           | Yes           |
 | HubSpot                                                                                             | No            |
 | Mailchimp                                                                                           | Yes           |
+| [MailerLite](/rownd/guides/configuration/integrations/mailerlite)                                   | Yes           |
```

Comment:
this link doesn't work. you might just want `./mailerlite`

Reply:
All of the links weren't working, so I fixed them!